src/convertToObject.js

```diff
@@ -3,16 +3,37 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObj = {};
+  const keyValueArray = [];
+  const stylesArray = sourceString.split(';');
+
+  function fillObj(obj, e) {
+    obj[e[0]] = e[1];
+  }
+
+  stylesArray.forEach(
+    x => x.trim().length > 2
+      ? keyValueArray.push(x.split(':'))
+      : null);
```

Comment:
1) no need to check if line length is greater than 2, checking for non-emptiness is enough
2) let's use `filter` method, you literally do the same job but in a twisted way